go/cli/mcap/cmd/sort.go

```diff
@@ -0,0 +1,157 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/foxglove/mcap/go/mcap"
+	"github.com/spf13/cobra"
+)
+
+var (
+	sortOutputFile string
+)
+
+type errUnindexedFile struct {
+	err error
+}
+
+func (e errUnindexedFile) Error() string {
+	return e.err.Error()
+}
+
+func (e errUnindexedFile) Is(tgt error) bool {
+	_, ok := tgt.(errUnindexedFile)
+	return ok
+}
+
+func sortFile(w io.Writer, r io.ReadSeeker) error {
+	reader, err := mcap.NewReader(r)
+	if err != nil {
+		return fmt.Errorf("failed to create reader: %w", err)
+	}
+	writer, err := mcap.NewWriter(w, &mcap.WriterOptions{
+		Chunked:     true,
+		Compression: mcap.CompressionZSTD,
+		ChunkSize:   4 * 1024 * 1024,
```

Comment:
should we make these settings configurable? (I think they are for some of the other subcommands, but can't recall)

Reply:
I really wish we just had sane writer defaults, then we wouldn't need to even think about this.. :(